### PR TITLE
Support int vid in mutate sentences

### PR DESCRIPTION
**Pull request by laura-ding**
Test with the pytest. here not add the test file.
TODO: need to write features file,  but the features file need another pr supported
https://github.com/vesoft-inc/nebula-graph/pull/475

### CODE REVIEW

File: src/parser/parser.yy [hunk 1]
```diff
@@ -996,9 +999,6 @@ vid
         $$ = new ConstantExpression(*$1);
         delete $1;
     }
-    | legal_integer {
```
Comment (Shylock-Hg) [inline src/parser/parser.yy:999]: Why change this?

Comment (laura-ding) [reply src/parser/parser.yy:999]: `legal_integer` is a positive integer, `int vid` can be a negative integer.

Comment (Shylock-Hg) [reply src/parser/parser.yy:999]: Ok.

**Review by nevermore3 — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Comment by yixinglu** [comment]
I don't think it's an improvement about `addDescription` changes. At most scenarios, we don't pass a `Value` to it, it's not necessary to do this conversion from string to Value and from Value to string again.

**Pull request closed without merging**